scripts/checkformat_checkpatch.sh

```diff
@@ -4,4 +4,4 @@ wget --no-clobber --quiet $linux/plain/scripts/spelling.txt
 wget --no-clobber --quiet $linux/plain/scripts/checkpatch.pl
 wget --no-clobber --quiet $linux/plain/scripts/const_structs.checkpatch
 chmod +x checkpatch.pl
-./checkpatch.pl --terse --no-tree --max-line-length=80 --ignore ARRAY_SIZE,VOLATILE,LEADING_SPACE --file src/*.[ch]
+./checkpatch.pl --terse --no-tree --max-line-length=80 --ignore ARRAY_SIZE,LEADING_SPACE,SPLIT_STRING,VOLATILE --file src/*.[ch]
```

Comment:
What is being ignored with SPLIT_STRING?

Reply:
String literals split in multiple lines as in `log_control_variables()`. :blush: